### ISSUE TITLE
Request for Training Logs & Loss Curves for Custom Data Finetuning

### ISSUE THREAD

**Issue by lnSong** [facebookresearch/map-anything#114]
Thank you for your excellent work!
Could you please provide the training logs to facilitate finetuning on custom data? Additionally, it would be greatly helpful if you could share the loss curves (i.e., how the loss evolves during training) and the final loss value after the model converges.

**Comment by Nik-V9** [comment]
Hi, these are the training curves for the new MapA V1.1. checkpoints corresponding to the bash scripts here: https://github.com/facebookresearch/map-anything/tree/main/bash_scripts/train/main

Unfortunately it's not easy for me to share the tensorboard logs. The below values are what I look at when training:

<img width="1015" height="593" alt="Image" src="https://github.com/user-attachments/assets/6784fb65-96a4-4d32-8302-25e18b81057d" />

<img width="1022" height="584" alt="Image" src="https://github.com/user-attachments/assets/7dcab436-26c8-4092-8cef-bbb04668bad5" />

<img width="1008" height="797" alt="Image" src="https://github.com/user-attachments/assets/713ff4c2-31dc-4304-83d1-1c992fb028d5" />

<img width="1018" height="893" alt="Image" src="https://github.com/user-attachments/assets/6edd3620-d790-42c5-b6aa-a2f89ccf8959" />

<img width="1014" height="432" alt="Image" src="https://github.com/user-attachments/assets/d4bc30f9-e0e2-435a-8211-8fcdfb334358" />

<img width="510" height="442" alt="Image" src="https://github.com/user-attachments/assets/9e5da9a4-dad5-4c8c-8136-83938e5fcdba" />

**Comment by lnSong** [comment]
Thank you for your enthusiastic assistance!
